dev-tools/ls-apis/tests/api_dependencies.out

```diff
@@ -70,6 +70,8 @@ Nexus Internal API (client: nexus-client)
     consumed by: oximeter-collector (omicron/oximeter/collector) via 1 path
     consumed by: propolis-server (propolis/bin/propolis-server) via 3 paths
 
+Nexus Internal Lockstep API (client: nexus-lockstep-client)
+
```

Comment:
Is it surprising that sled agent (for RSS) doesn't show up here?  Or is that because we haven't moved stuff over yet?